src/Node.java

```diff
@@ -0,0 +1,26 @@
+import java.util.HashMap;
+
+/**
+ * Created by Alberto on 3/30/2017.
+ */
+public class Node {
+    long xCoord;    //X coordinate
+    long yCoord;    //Y coordinate
+    String id;      //Name of the Node
```

Comment:
This is what we had in the diagram but it should really be a long or hash.  Our bad -Lead Devs :(